themes/ddbasic/sass/components/node/paragraphs.scss

```diff
@@ -0,0 +1,171 @@
+//
+// Styling of paragraphs
+
+@import '../../base.scss';
+
+// Paragraphs block display
+.paragraphs-block {
+  position: relative;
+  display: inline-block;
+  margin: 0 0 $distance-paragraph;
+  width: 100%;
+
+  &.half_left {
+    float: left;
+    padding-right: 3%;
+    width: 50%;
```

Comment:
This uses floats to achieve side-by-side layout. Further down we use flexbox which is a lot more modern. Is there a reason why this is not used here?

Reply:
I don't believe these are mutually exclusive.
Float allows text to wrap around the boxes, which we want from the outer wrappers.
https://jsfiddle.net/zq915kuw/

Flexbox allows use of columns within the boxes, like a bit of text next to an image and is the preferred approach in those situations